walkabout-style.yaml

```diff
@@ -468,6 +468,12 @@ fonts:
         - weight: normal
           #url: https://fonts.gstatic.com/s/montserrat/v7/zhcz-_WihjSQC0oHJ9TCYL3hpw3pgy2gAi-Ip7WPMi0.woff
           url: fonts/Montserrat-Regular.woff
+        - weight: 500
+          #url: https://fonts.gstatic.com/s/montserrat/v10/BYPM-GE291ZjIXBWrtCweqLy3K7m9xj_O5xwN3SdQNs.woff
+          url: fonts/Montserrat-Medium.woff
+        - weight: 700
```

Comment:
Please check other house styles to see if they need the same fix (new font def, new font file).

Reply:
Refill and Tron now also have changes applied, others were good (with changes around original comment).